project/dexter-server/public/AdminSE/adminSECtrl.js

```diff
@@ -0,0 +1,85 @@
+adminSEApp.controller('adminSECtrl', function($scope, $http, $location ,$routeParams, $log) {
+    "use strict";
+
+    $('select').on('select2:select', function (e) {
```

Comment:
Select2 gives you a customizable select box with support for searching, tagging, remote data sets, infinite scrolling, and many other highly used options.
https://select2.github.io/